src/epoch.rs

```diff
@@ -28,8 +28,8 @@ use pyo3::prelude::*;
 #[cfg(feature = "python")]
 use pyo3::pyclass::CompareOp;
 
-#[cfg(feature = "std")]
-use serde::{de, Deserialize, Deserializer};
+#[cfg(feature = "serde")]
```

Comment:
```suggestion
#[cfg(feature = "std")]
```

Reply:
so in conclusion, "serde" already exist in the current Cargo.toml
and it's usually invoked with "serde"